### DISCUSSION TITLE
Only prints `log_data` if `LOG_LEVEL` == `DEBUG`

### DISCUSSION THREAD

**Discussion by pedrounes1** [tiangolo/uvicorn-gunicorn-docker#228] [Questions] [answered]
I think that container only needs to print `log_data` from `docker-image/gunicorn_conf.py` if `LOG_LEVEL` is set up as `DEBUG`, otherwise there is no need for that.

Is there a way to supress that print that am I missing?

**Comment by tiangolo** [comment, marked as answer]
I'm not sure I understand the question :thinking: 

But anyway, just so you know, now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-docker#-warning-you-probably-dont-need-this-docker-image

And then the configuration will probably be much simpler as well, just dealing with Uvicorn and not with Gunicorn. :sunglasses:
